app/frontend/src/features/communities/constants.ts

```diff
@@ -14,6 +15,10 @@ export const CLOSE = "Close";
 export const COMMENT = "Comment";
 export const COMMENTS = "Comments";
 export const COMMUNITY_HEADING = (name: string) => `Welcome to ${name}!`;
+export const COMMUNITY_LEADERS = "Community leaders";
+export const COMMUNITY_LEADERS_DESCRIPTION =
+  "Below is a list of current leaders within this community.";
```

Comment:
Is this from the desktop mockup? "Leaders" sounds pretty weird to me...

Reply:
Yeah, and then it has another heading on "Community moderators" right underneath that. Maybe I should move "Community moderators" into the h2 title and remove this description? It's kinda obvious from looking at the UI with what the title says right?

Reply:
Yes definitely